corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -461,6 +461,15 @@ def _get_form_ids_for_user(domain, user_id, is_deleted):
             results = fetchall_as_namedtuple(cursor)
             return [result.form_id for result in results]
 
+    @staticmethod
+    @transaction.atomic
+    def write_blob_bucket(attachment, bucket):
+        with get_cursor(XFormInstanceSQL) as cursor:
+            cursor.execute(
+                'SELECT write_blob_bucket(%s, %s::text)',
```

Comment:
Surprised that you need to do this since the cursor doesn't know anything about the model here and the value being passed is just a string and not a field object or anything like that.